utils/transactionExecutor.ts

```diff
@@ -120,33 +120,21 @@ export const estimate = async (
   return info.partialFee.toString()
 }
 
-const estimateEvm = async ({ address, arg, abi, functionName, account, prefix }: ExecuteEvmTransactionParams & { account: string, prefix: Prefix }) => {
-  const { publicClient } = useViem(prefix)
-
-  const [estimatedGas, gasPrice] = await Promise.all([
-    publicClient.estimateContractGas({
-      account: account as Address,
-      address: address as Address,
-      abi,
-      args: arg,
-      functionName,
-    }),
-    publicClient.getGasPrice(),
-  ])
-
-  return String(estimatedGas * gasPrice)
+const estimateEvm = async () => {
+  const wagmiConfig = useConfig()
+  const gasPrice = await getGasPrice(wagmiConfig)
+
+  return String(gasPrice)
```

Comment:
it's not the same

Reply:
Now you are just asking for the gas price

https://wagmi.sh/core/api/actions/estimateGas#estimategas